dotnetapp-selfcontained/README.md

```diff
@@ -1,42 +1,48 @@
 dotnetapp-selfcontained Sample
```

Comment:
Should we be including all supported rids in the project.json so that if someone is running on a supported distro the steps below will work?